python/setup.py

```diff
@@ -37,6 +37,11 @@
     "ray/autoscaler/aws/example-full.yaml"
 ]
 
+if "RAY_USE_NEW_GCS" in os.environ and os.environ["RAY_USE_NEW_GCS"] == "on":
```

Comment:
I think 1/0 is more standard than "on"/"off"

Reply:
Ok yeah, but I prefer to do it as a separate PR since it also touches other code and tests that are already checked in